examples/async_pipeline.ipynb

```diff
@@ -0,0 +1,401 @@
+{
```

Comment:
<p>"pipeline connections the same"</p><p> </p><p>missing "are"?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-experimental/pull/142/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>